pandas/tests/test_strings.py

```diff
@@ -1153,6 +1153,12 @@ def test_repeat(self):
         assert isinstance(rs, Series)
         tm.assert_series_equal(rs, xp)
 
+        # GH: 31632
```

Comment:
can you make a new test (call it test_repeat_with_null)

Reply:
Done.